app/lib/frontend/handlers.dart

```diff
@@ -422,13 +422,10 @@ Future<shelf.Response> packageVersionHandlerHtml(
       final Stopwatch serviceSw = new Stopwatch()..start();
       final AnalysisData analysisData = await analyzerClient.getAnalysisData(
           selectedVersion.package, selectedVersion.version);
-      final AnalysisView analysisView = (analysisData != null &&
-              analysisData.analysisStatus != AnalysisStatus.aborted)
-          ? new AnalysisView(analysisData)
+      final AnalysisView analysisView = new AnalysisView(analysisData);
+      analysisTabContent = analysisView.hasAnalysisData
+          ? templateService.renderAnalysisTab(analysisView)
```

Comment:
Since `renderAnalysisTab` returns `null` now, you can remove the condition here.